### PR TITLE
Accessibility statement v1

### PR DESCRIPTION
**Pull request by kkoskelin**
First draft of accessibility statement; includes route, view, sequence
Screen Shot 2020-03-02 at 9 39 51 AM

Currently this view is NOT linked from anywhere in the application but is available on the route `/accessibility-statement`

### CODE REVIEW

File: web-client/src/presenter/sequences/gotoAccessibilityStatementSequence.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { setCurrentPageAction } from '../actions/setCurrentPageAction';
+
+export const gotoAccessibilityStatementSequence = [
```
Comment (matthopson) [inline web-client/src/presenter/sequences/gotoAccessibilityStatementSequence.js:3]: "goto statement" 🤣

Comment (kkoskelin) [reply web-client/src/presenter/sequences/gotoAccessibilityStatementSequence.js:3]:
```
10 print "Matt is Awesome"
20 goto 10
```

File: web-client/src/views/Accessibility/AccessibilityStatement.jsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+      <span className="basic-information conformance-standard">
+        <span data-negate="">WCAG 2.0 level AA</span>.{' '}
+      </span>
+      <span>
+        <span className="basic-information conformance-status">
+          Fully conformant
+        </span>{' '}
+        means that{' '}
+        <span className="basic-information conformance-meaning">
+          the content fully conforms to the accessibility standard without any
```
Comment (matthopson) [inline web-client/src/views/Accessibility/AccessibilityStatement.jsx:72]: Is AA considered "fully conformant without exception"?

**Review by matthopson — APPROVED** [review]

File: web-client/src/views/Accessibility/AccessibilityStatement.jsx (hunk 2 above)
Comment (kkoskelin) [reply web-client/src/views/Accessibility/AccessibilityStatement.jsx:72]: To the best of our knowledge, our application conforms to WCAG2AA without any exceptions. -- for the AA-level standard.  So... yes? (the same could not be said for AAA-level, but that is not the standard we are attempting to adhere to)

**Pull request merged**